unity-client/Assets/Scripts/MainScripts/DCL/Helpers/UIHelpers/DynamicScrollSensitivity.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// Let to set a dynamic sensitivity value depending on the height of the content container.
+/// Call to RecalculateSensitivity() function to calculate and apply the new sensitivity value.
+/// </summary>
+public class DynamicScrollSensitivity : MonoBehaviour
+{
+    [Tooltip("Scroll Rect component that will be modified.")]
+    public ScrollRect mainScrollRect;
+    [Tooltip("Viewport associated to the Scroll Rect.")]
+    public RectTransform viewport;
+    [Tooltip("Transform that will contain all the items of the viewport.")]
+    public RectTransform contentContainer;
+    [Tooltip("Min value for the calculated scroll sensitivity.")]
+    public float minSensitivity = 10f;
+    [Tooltip("Max value for the calculated scroll sensitivity.")]
+    public float maxSensitivity = 100f;
```

Comment:
We could use a Curve here, to allow further customization

Reply:
I don't know if it's worth to use an AnimationCurve in this case, because the sensitivity calculation is not only depends on these 2 values (`minSensitivity` and `maxSensitivity`) but it also depends on the current heights of the content and viewport gameobjects:
`float newSensitivity = contentContainer.rect.height * minSensitivity / viewport.rect.height;`

Reply:
Fair enough!